docs/src/main/asciidoc/security-overview-concept.adoc

```diff
@@ -438,7 +193,3 @@ You can optionally apply a time limit to individual suppressions by adding an ex
 `<suppress until="2022-01-01Z">...</suppress>`
 You can adjust the expiry date if you need to.
 
-== Quarkus Security testing
```

Comment:
This H2 section is moved further up within this overview topic. (Eventually this could evolve into a bigger how-to / tutorial.)

Reply:
FYI, we already have a security-testing.adoc, this section is only meant to link to it